### ISSUE TITLE
Always parse recordings

### ISSUE THREAD

**Issue by jesus2099** [jesus2099/konami-command#270]
Remove the _There are XXX RECORDINGS to parse on this page_ warning prompt and always parse as the longest job has already been done at this point (as we can already see we do have XXX recordings).
Maybe an effort on the DOM manipulation could be checked (build nodes and only once change the DOM).

**Comment by jesus2099** [comment]
Already done for COLLECTION HIGHLIGHTER in 38a86b89a50736be206a893a26197ee06f00f718

**Comment by jesus2099** [comment]
OK apparently done, already.
I must have forgot to close the ticket.

**Comment by jesus2099** [comment]
Now also done for COOL ENTITY LINKS in ed1debce954a917d071050bdc81897793508c1b4 / https://github.com/jesus2099/konami-command/commit/ed1debce954a917d071050bdc81897793508c1b4?w=1
